tests/functional/repl.sh

```diff
@@ -162,15 +162,32 @@ foo + baz
 # - Modify the flake
 # - Re-eval it
 # - Check that the result has changed
-replResult=$( (
-echo "changingThing"
-sleep 1 # Leave the repl the time to eval 'foo'
+mkfifo repl_fifo
+nix repl ./flake --experimental-features 'flakes' < repl_fifo > repl_output 2>&1 &
+repl_pid=$!
+exec 3>repl_fifo # Open fifo for writing
+echo "changingThing" >&3
+for i in $(seq 1 100); do
```

Comment:
I think a hundred-ish seconds is still reasonable, and reduces the chance of this triggering when testing on a slow machine with high load.

```suggestion
for i in $(seq 1 1000); do
```